mobile/src/main/java/org/openhab/habdroid/core/message/MessageHandler.java

```diff
@@ -0,0 +1,87 @@
+package org.openhab.habdroid.core.message;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.content.DialogInterface;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
+import android.support.design.widget.Snackbar;
+import android.util.Log;
+import android.widget.Toast;
+
+import org.openhab.habdroid.util.Constants;
+
+public class MessageHandler {
+    private static final String TAG = MessageHandler.class.getSimpleName();
+
+    /**
+     * Shows a message to the user.
+     * You might want to send two messages: One detailed one with
+     * logLevel Constants.MESSAGES.LOGLEVEL.DEBUG and one simple message with
+     * Constants.MESSAGES.LOGLEVEL.NO_DEBUG
```

Comment:
I think you're right. `Constants.MESSAGES.LOGLEVEL.NO_DEBUG` is a bit too long. Can you move the message constants to this file and name them `LOGLEVEL_DEBUG` and `TYPE_DIALOG`?

Reply:
Thats my answer to https://github.com/openhab/openhab.android/pull/389#discussion_r142097574

Reply:
Done! :)